@navikt/core/css/tooltip.css

```
@@ -0,0 +1,73 @@
+@keyframes tooltipFadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+.navds-tooltip {
+  z-index: 2147483647;
+}
+
+.navds-tooltip__inner {
+  background-color: var(--navds-semantic-color-component-background-inverted);
+  color: var(--navds-semantic-color-text-inverted);
+  border-radius: var(--navds-border-radius-medium);
+  padding: 2px var(--navds-spacing-2);
+  align-items: center;
+  filter: drop-shadow(0 1px 2px rgba(0, 0, 0, 0.05))
+    drop-shadow(0 1px 3px rgba(0, 0, 0, 0.1));
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+  min-width: 8rem;
+  animation-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  animation-duration: 0.2s;
+  animation-name: tooltipFadeIn;
+}
+
+.navds-tooltip__arrow {
+  height: 0.5rem;
+  width: 0.5rem;
+  transform: rotate(45deg);
+  z-index: -1;
+  background-color: var(--navds-semantic-color-component-background-inverted);
+  position: absolute;
+}
+
+.navds-tooltip--inverted > .navds-tooltip__inner {
+  background-color: var(--navds-semantic-color-component-background-light);
+  color: var(--navds-semantic-color-text);
+}
+
+.navds-tooltip--inverted > .navds-tooltip__inner .navds-tooltip__arrow {
+  background-color: var(--navds-semantic-color-component-background-light);
+}
+
+.navds-tooltip__keys {
+  padding-bottom: 0.125rem;
+}
+
+.navds-tooltip__key {
+  font-family: var(--navds-font-family);
+  background-color: var(--navds-global-color-gray-700);
+  color: var(--navds-semantic-color-text-inverted);
+  border-radius: 2px;
+  padding: 0 4px;
+  min-width: 18px;
+  height: 18px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.navds-tooltip__key + .navds-tooltip__key {
+  margin-left: 0.125rem;
+}
+
+.navds-tooltip__key--inverted {
```

Kan vi gjøre noe sånt, og så fjerne `navds-tooltip__key--inverted`-klassen?
```suggestion
.navds-tooltip--inverted .navds-tooltip__key {
```

---

👍